src/components/layout/Quiz/StateQuestion.js

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import { connect } from "react-redux"
+import PropTypes from "prop-types"
+
+// Redux
+import { setLocation } from "../../../actions/quizActions"
+
+// Material UI
+import {
+  FormControl,
+  InputLabel,
+  makeStyles,
+  MenuItem,
+  Select,
+  Typography,
+} from "@material-ui/core"
+
+const STATES = [
+  "AL",
+  "AK",
+  "AZ",
+  "AR",
+  "CA",
+  "CO",
+  "CT",
+  "DE",
+  "FL",
+  "GA",
+  "HI",
+  "ID",
+  "IL",
+  "IN",
+  "IA",
+  "KS",
+  "KY",
+  "LA",
+  "ME",
+  "MD",
+  "MA",
+  "MI",
+  "MN",
+  "MS",
+  "MO",
+  "MT",
+  "NE",
+  "NV",
+  "NH",
+  "NJ",
+  "NM",
+  "NY",
+  "NM",
+  "NY",
+  "NC",
+  "ND",
+  "OH",
+  "OK",
+  "OR",
+  "PA",
+  "RI",
+  "SC",
+  "SD",
+  "TN",
+  "TX",
+  "UT",
+  "VT",
+  "VA",
+  "WA",
+  "WV",
+  "WI",
+  "WY",
+]
+
+const useStyles = makeStyles(theme => ({
+  formControl: {
+    minWidth: 200,
+    marginTop: 10,
+    marginLeft: 20,
+  },
+}))
+
+const StateQuestion = ({ quiz, setLocation }) => {
+  const classes = useStyles()
+
+  const handleChange = e => {
+    setLocation(e.target.value)
+  }
+
+  return (
+    <>
+      <Typography variant="h2">Question 1: Location</Typography>
+
+      <FormControl className={classes.formControl}>
+        <InputLabel id="state">Where do you live?</InputLabel>
+        <Select
+          labelId="state"
```

Comment:
For some reason, React is printing console warnings around this prop, even though it's clearly correct Material UI usage.

```
Warning: React does not recognize the `labelId` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `labelid` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

We don't have to solve this right now, but we should figure out how to silence this some time soon so that we can keep our console warnings under control.

Reply:
Added an inline TODO.